instructure/canvas-ios
# Module item sequence

refs: MBL-14137
affects: student
release note: none

To do:
- [x] Merge with master
- [x] Prettier next/prev buttons
- [x] Better External Tool details
- [x] Unit tests for all the types

Test plan:
- As a student, keep the student_modules feature flag off and view modules in a course all the module types (s > CS 1400 > Modules). With the flag off, everything should be unchanged in Student app
- As a student, enable the student_modules feature flag and view modules
- Each module item type should load in the module sequence view with the next/prev buttons
- View a module item from a deep link in Page (s > CS 1400 > Page > Links to module items)
- View a file/assignment/discussion/quiz/page from their respective lists. If they belong to a module they should load in the sequence view with next/prev. If they do not belong to a module, the sequence buttons should not show
- As a teacher in Teacher app, module items (from the Modules list) should load in the sequence view (even with the flag disabled). This should not cause any issues.  When the user progresses with next/previous buttons, the item should become selected in the list on iPad.
